File: Colleen.c
#include <stdio.h>

/*
	Comment0
*/

void f(void){}

int main(void){
/*
	Comment1
*/
	f();
	char *c="#include <stdio.h>%c%1$c/*%1$c	Comment0%1$c*/%1$c%1$cvoid f(void){}%1$c%1$cint main(void){%1$c/*%1$c	Comment1%1$c*/%1$c	f();%1$c	char *c=%2$c%3$s%2$c;%1$c	printf(c, 10, 34, c);%1$c}";
	printf(c, 10, 34, c);
}

File: php/Colleen.php
#!/usr/bin/php
<?PHP
/*
	Comment0
*/

function f(){}

function main(){
/*
	Comment1
*/
	f();
	$c="#!/usr/bin/php%c<?PHP%c/*%c	Comment0%c*/%c%cfunction f(){}%c%cfunction main(){%c/*%c	Comment1%c*/%c	f();%c	%cc=%c%s%c;%c	printf(%cc, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 36, 34, %cc, 34, 10, 36, 36, 10, 10, 10);%c}%cmain();%c?>";
	printf($c, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 36, 34, $c, 34, 10, 36, 36, 10, 10, 10);
}
main();
?>